### DISCUSSION TITLE
RE: Signifcant but have "+0" change

### DISCUSSION THREAD

**Discussion by Malabady** [hahnlab/CAFE5#176] [Q&A] [answered]
Hello:

A number of the gene families are significant (Base, p=0.004), but the number of gene change is "+0" in the  base change file. Is this normal? if yes, what does it mean? 
Here is an example:
from Base_family_results.txt:

OG0000096       0.004   y

From Base_change.tab:
<!DOCTYPE html>
<html>
<head>

</head>

<body>

FamilyID | My Species | Species A | Species B | Species C | Species D | <5> | Species E | <7> | <8> | <9> | Species F | <11> | Speices G | <13> | Species H | <15> | Species I | <17>
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
OG0000096 | 0 | 4 | -6 | -3 | 1 | 1 | 5 | -2 | 0 | 0 | 2 | 1 | -2 | 1 | 8 | 0 | -5 | 0

</body>

</html>

Thank you for the help,

**Comment by hahnlab-user** [comment, marked as answer]
I think the p-value you have asked for is for the whole tree, not branch-by-branch. So CAFE is telling you this family is significant, but not on any particular branch.
